archivebox/core/models.py

```diff
@@ -181,7 +181,7 @@ class ArchiveResult(models.Model):
     snapshot = models.ForeignKey(Snapshot, on_delete=models.CASCADE)
     cmd = JSONField()
     pwd = models.CharField(max_length=256)
-    cmd_version = models.CharField(max_length=32)
+    cmd_version = models.CharField(max_length=32, default="")
```

Comment:
```suggestion
    cmd_version = models.CharField(max_length=32, null=True, blank=True, default=None)
```
I'd rather not store emptystrings if possible.

Reply:
Ok, changed.